bundle.Dockerfile

```diff
@@ -7,7 +7,7 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=node-healthcheck-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=candidate
 LABEL operators.operatorframework.io.bundle.channel.default.v1=candidate
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.14.0+git
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.18.0+git
```

Comment:
Should be 1.17- see [Michael's comment](https://github.com/medik8s/self-node-remediation/pull/6/files#r842779700).

Reply:
It is fine, I haven't noticed there is a new version of Operator-sdk (see [1.19](https://github.com/operator-framework/operator-sdk/releases/tag/v1.19.0)).

Reply:
I uodated to sdk 1.19...